testcafe/tests/photos/create_full_album_scenario.js

```diff
@@ -1,96 +1,211 @@
-  await photoAlbumPage.addPhotosToAlbum(`New2_${ALBUM_DATE_TIME}`, 3, 2)
+  await photoAlbumsPage.goToAlbum(`New2_${FEATURE_PREFIX}`)
+  await photoAlbumPage.addPhotosToAlbum(`New2_${FEATURE_PREFIX}`, 3, 2)
   await photoAlbumPage.backToAlbumsList()
-  await photoAlbumsPage.isAlbumExistsAndVisible(`New2_${ALBUM_DATE_TIME}`, 5)
+  await photoAlbumsPage.isAlbumExistsAndVisible(`New2_${FEATURE_PREFIX}`, 5)
   console.groupEnd()
 })
 
-test('Go to New2_ALBUM_DATE_TIME, and remove the 1st photos', async () => {
-  console.group(
-    `↳ ℹ️  Go to New2_${ALBUM_DATE_TIME}, and remove the 1st photos`
+test(TEST_ADD_EXISTING_PHOTOS, async t => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_ADD_EXISTING_PHOTOS}`)
+  await timelinePage.selectPhotos(1)
+  await timelinePage.checkCozyBarOnTimeline()
+  await t.click(selectors.btnAddToAlbumCozySelectionBar)
+
+  await t.expect(selectors.spinner.exists).notOk('Spinner still spinning')
+  await checkAllImagesExists()
+
+  await t.fixtureCtx.vr.takeScreenshotAndUpload({
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_ADD_EXISTING_PHOTOS}-1`,
+    withMask: maskPhotosCluster
+  })
+  await t.click(selectors.album(`New2_${FEATURE_PREFIX}`))
+
+  //Wait for toast alert to disapear before taking screenshot
+  await checkToastAppearsAndDisappears(
+    'This album already contains this photo.'
   )
+  //go to album to get the screenshot
   await timelinePage.goToAlbums()
-  await photoAlbumsPage.goToAlbum(`New2_${ALBUM_DATE_TIME}`)
-  await photoAlbumPage.removePhoto(1)
+  await photoAlbumsPage.goToAlbum(`New2_${FEATURE_PREFIX}`)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload({
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_ADD_EXISTING_PHOTOS}-2`,
+    delay: THUMBNAIL_DELAY,
+    pageToWait: photoAlbumPage
+  })
   console.groupEnd()
 })
 
-test('Go to New2_ALBUM_DATE_TIME, and delete it', async () => {
-  console.group(`↳ ℹ️  Go to New2_${ALBUM_DATE_TIME}, and delete it`)
+test(TEST_DELETE_PHOTO_FROM_TIMELINE, async t => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_DELETE_PHOTO_FROM_TIMELINE}`)
+  await timelinePage.selectPhotosByName([IMG0])
+  await timelinePage.deletePhotosFromTimeline({
+    numOfFiles: 1,
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_DELETE_PHOTO_FROM_TIMELINE}-1`,
+    withMask: maskPhotosCluster
+  })
+  await t.fixtureCtx.vr.takeScreenshotAndUpload({
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_DELETE_PHOTO_FROM_TIMELINE}-2`,
+    delay: THUMBNAIL_DELAY,
+    pageToWait: timelinePage,
+    withMask: maskPhotosCluster
+  })
+  console.groupEnd()
+})
+
+test(TEST_REMOVE_PHOTO, async t => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_REMOVE_PHOTO}`)
   await timelinePage.goToAlbums()
-  await photoAlbumsPage.goToAlbum(`New2_${ALBUM_DATE_TIME}`)
+  await photoAlbumsPage.goToAlbum(`New2_${FEATURE_PREFIX}`)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload({
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_REMOVE_PHOTO}-1`,
```

Comment:
I think that instead of `-1`/`-2`, descriptive names for screenshots would be helpful when we have screenshots to validate during review. I think we shouldn't have to look at the test's code to be able to understand if the screenshot is ok. Something like `${FEATURE_PREFIX}/${TEST_REMOVE_PHOTO}-before` and `${FEATURE_PREFIX}/${TEST_REMOVE_PHOTO}-after` are good if `TEST_REMOVE_PHOTO` is "remove first photo from album".